network/src/peer_manager/mod.rs

```diff
@@ -365,13 +365,16 @@ where
             .with_label_values(&[role, "connected"])
             .set(total as i64);
 
-        counters::LIBRA_NETWORK_PEERS
-            .with_label_values(&[role, "inbound"])
-            .set(inbound as i64);
-
-        counters::LIBRA_NETWORK_PEERS
-            .with_label_values(&[role, "outbound"])
-            .set(outbound as i64);
```

Comment:
is this counter still being used? if not lets delete it

 we'll need to update dashboards.

Reply:
It is probably better to either rebase this onto the premainnet branch or to get it landed in master

Reply:
The original counter is the one that's use for all the upstream peer information, and on dashboards.